Notebook CI test fails with `kernel died`

### Description

The following error is currently raised in the `test_notebook` CI job:

```
[NbConvertApp] ERROR | Kernel died while waiting for execute reply.
[21](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:22)
Traceback (most recent call last):
[22](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:23)
  File "/usr/share/miniconda3/envs/bitinfo/bin/jupyter-nbconvert", line 10, in <module>
[23](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:24)
    sys.exit(main())
[24](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:25)
             ^^^^^^
[25](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:26)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/jupyter_core/application.py", line 277, in launch_instance
[26](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:27)
    return super().launch_instance(argv=argv, **kwargs)
[27](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:28)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[28](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:29)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/traitlets/config/application.py", line 1043, in launch_instance
[29](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:30)
    app.start()
[30](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:31)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/nbconvertapp.py", line 414, in start
[31](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:32)
    self.convert_notebooks()
[32](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:33)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/nbconvertapp.py", line 589, in convert_notebooks
[33](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:34)
    self.convert_single_notebook(notebook_filename)
[34](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:35)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/nbconvertapp.py", line 555, in convert_single_notebook
[35](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:36)
    output, resources = self.export_single_notebook(
[36](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:37)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[37](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:38)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/nbconvertapp.py", line 481, in export_single_notebook
[38](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:39)
    output, resources = self.exporter.from_filename(
[39](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:40)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[40](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:41)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/exporters/templateexporter.py", line 388, in from_filename
[41](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:42)
    return super().from_filename(filename, resources, **kw)  # type:ignore
[42](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:43)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[43](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:44)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/exporters/exporter.py", line 201, in from_filename
[44](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:45)
    return self.from_file(f, resources=resources, **kw)
[45](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:46)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[46](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:47)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/exporters/templateexporter.py", line 394, in from_file
[47](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:48)
    return super().from_file(file_stream, resources, **kw)  # type:ignore
[48](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:49)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[49](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:50)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/exporters/exporter.py", line 220, in from_file
[50](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:51)
    return self.from_notebook_node(
[51](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:52)
           ^^^^^^^^^^^^^^^^^^^^^^^^
[52](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:53)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/exporters/html.py", line 234, in from_notebook_node
[53](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:54)
    return super().from_notebook_node(nb, resources, **kw)
[54](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:55)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[55](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:56)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/exporters/templateexporter.py", line 410, in from_notebook_node
[56](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:57)
    nb_copy, resources = super().from_notebook_node(nb, resources, **kw)
[57](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:58)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[58](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:59)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/exporters/exporter.py", line 154, in from_notebook_node
[59](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:60)
    nb_copy, resources = self._preprocess(nb_copy, resources)
[60](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:61)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[61](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:62)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/exporters/exporter.py", line 352, in _preprocess
[62](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:63)
    nbc, resc = preprocessor(nbc, resc)
[63](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:64)
                ^^^^^^^^^^^^^^^^^^^^^^^
[64](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:65)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/preprocessors/base.py", line 48, in __call__
[65](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:66)
    return self.preprocess(nb, resources)
[66](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:67)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[67](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:68)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/preprocessors/execute.py", line 100, in preprocess
[68](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:69)
    self.preprocess_cell(cell, resources, index)
[69](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:70)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbconvert/preprocessors/execute.py", line 121, in preprocess_cell
[70](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:71)
    cell = self.execute_cell(cell, index, store_history=True)
[71](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:72)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[72](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:73)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
[73](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:74)
    return loop.run_until_complete(inner)
[74](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:75)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[75](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:76)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
[76](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:77)
    return future.result()
[77](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:78)
           ^^^^^^^^^^^^^^^
[78](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:79)
  File "/usr/share/miniconda3/envs/bitinfo/lib/python3.11/site-packages/nbclient/client.py", line 1005, in async_execute_cell
[79](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:80)
    raise DeadKernelError("Kernel died") from None
[80](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:81)
nbclient.exceptions.DeadKernelError: Kernel died
[81](https://github.com/observingClouds/xbitinfo/actions/runs/5229009356/jobs/9441730211#step:8:82)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -101,6 +101,9 @@ jobs:
       - name: Set up conda environment
         run: |
           mamba env update -f environment.yml
+      - name: Remove julia (issue #212)
+        run: |
+          conda remove julia
       - name: Conda info
         run: conda info
       - name: Conda list

